src/adm/index.ts

```diff
@@ -80,8 +81,46 @@ const router = createRouter({
         }),
       },
     },
-    handler: async (request) =>
+    handler: async (request: Request) =>
       Response.json(await blockUser(await request.json())),
+  })
+  .route({
+    method: 'POST',
+    path: '/moderation/awardBadge',
```

Comment:
請按照[之前的設計](https://g0v.hackmd.io/@cofacts/rd/%2Fzx_Au6iiRN601tnMK7gx3A#API)，放在 `/badge/award`
```suggestion
    path: '/badge/award',
```